test/hardhat/Router.test.ts

```diff
@@ -765,6 +780,102 @@ describe("Router", function () {
     const version = await router.getVersion();
     expect(version).to.equal("1.0.0");
   });
+
+  it("should revert if setSwapRequestBlsValidator is called with zero address", async () => {
+    const invalidAddress = ZeroAddress;
+    const currentNonce = Number(await router.currentNonce()) + 1;
```

Comment:
ditto for nonce stuff (optional)
```suggestion
    const currentNonce = Number(await router.currentNonce());
```